Megaparsec/Megaparsec.lean

```diff
@@ -234,20 +234,21 @@ def msₜ [m: Monad M]: Monad (StateT σ M) :=
 def asₜ [Monad A] [Alternative A]: Alternative (StateT σ A) :=
   StateT.instAlternativeStateT
 
-instance (E S : Type) [m: Monad M] [a: Alternative M] [s: Stream S] [mₚ: @MonadParsec M E S m a s] : @MonadParsec (StateT σ M) E S (@msₜ M σ m) (@asₜ M σ m a) s where
-  parseError A err := liftM (mₚ.parseError A err)
-  label A f st := (mₚ.label E S (A × σ) f) ∘ st
-  attempt A st := mₚ.attempt E S (A × σ) ∘ st
-  lookAhead A state x := mₚ.lookAhead E S (A × σ) (state x)
-  notFollowedBy A state x := seqComp (mₚ.notFollowedBy E S A (fst <$> state x)) (pure (Unit.unit , x))
+instance {E S : Type} [m: Monad M] [a: Alternative M] {s: Stream S} [mₚ: @MonadParsec M E S m a s] : @MonadParsec (StateT σ M) E S (@msₜ M σ m) (@asₜ M σ m a) s where
+  parseError A err := liftM (mₚ.parseError A)
+  label A f st := (mₚ.label E s (A × σ) f) ∘ st
+  attempt A st := mₚ.attempt E s (A × σ) ∘ st
+  lookAhead A state x := mₚ.lookAhead E s (A × σ) (state x)
+  notFollowedBy A state x := seqComp (mₚ.notFollowedBy E s A (fst <$> state x)) (pure (Unit.unit , x))
+  -- withRecovery A cont state x := mₚ.withRecovery (A × σ) (fun e => (cont e) x) (state x)
   withRecovery A cont state x := mₚ.withRecovery (A × σ) (fun e => (cont e) x) (state x)
```

Comment:
Why did you comment on `withRecovery` and left the same recovery on the next line?